src/HomeWork_9_Participant/Human.java

```diff
@@ -0,0 +1,40 @@
+package HomeWork_9_Participant;
+
+public class Human extends Participant{
+    public Human(String name, double runningDistance, double jumpHeight) {
+        super(name, runningDistance, jumpHeight);
+    }
+
+    @Override
+    void run(Obstacle obstacle) {
+        if (!(obstacle instanceof RunningTrack)){
+            return;
+        }
+        System.out.println(getName() + " " + obstacle.overcome() +
+                "s the track " + obstacle.getLenth() + " meters long");
+    }
+
+    @Override
+    void jump(Obstacle obstacle) {
+        if (!(obstacle instanceof Wall)){
+            return;
+        }
+        System.out.println(getName() + " " + obstacle.overcome() + " the wall "
+                + obstacle.getHight() + " meters long");
+    }
+
+    @Override
+    public String getName() {
```

Comment:
will write only one but it is related to all the getters and setters.  getters and setters are only attacked by the class directly, and if you override getters and setters from the parent object it will violate the encapsulation. So if you 
create getters and setters they need to be declared in the same class as the variable they return.